opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -55,3 +55,5 @@
     ],
     ...,
 ]
+
+ExtendedAttributes = Mapping[str, "AnyValue"]
```

Comment:
Any concerns about releasing this as public straight away? We could start with `_ExtendedAttributes` and go from there, especially since users don't need this named type.

Reply:
Fine for me